### PR TITLE
add coverage report to travis script

### LINKED ISSUE #268: Calculate current % unit test coverage
<bountysource-plugin>

---
Want to back this issue? **[Place a bounty on it!](https://www.bountysource.com/issues/6426336-calculate-current-unit-test-coverage?utm_campaign=plugin&utm_content=tracker%2F2451736&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F2451736&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by dduugg**
resolves #268

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,32 +3,23 @@ TESTS=tests/*.js
 MONGO_CONNECTION?=mongodb://localhost/test_db
 CUSTOMCONNSTR_mongo_settings_collection?=test_settings
 CUSTOMCONNSTR_mongo_collection?=test_sgvs
-
-BLANKET=--require blanket 
+MONGO_SETTINGS=MONGO_CONNECTION=${MONGO_CONNECTION} \
```
Comment (dduugg) [inline Makefile]: This looks sketchy but seems to work. I'm trying to keep the code dry since `test` and `travis-cov` use the same mongo settings.

File: README.md [hunk 2]
```diff
@@ -3,9 +3,10 @@ cgm-remote-monitor (a.k.a. Nightscout)
 
 [![Build Status][build-img]][build-url]
 [![Dependency Status][dependency-img]][dependency-url]
+[![Coverage Status][coverage-img]][coverage-url]
 [![Gitter chat][gitter-img]][gitter-url]
-[![Stories in Ready][ready-img]][ready-url]
-[![Stories in Progress][progress-img]][progress-url]
+[![Stories in Ready][ready-img]][waffle]
```
Comment (dduugg) [inline README.md]: An unrelated local cleanup that I'm happy to punt to another PR (`ready-url` and `progress-url` are the same).

Comment (jasoncalabrese) [inline README.md]: I kind of liked seeing the ready and in progress counts, the target url is the same but the counts are different.

Comment (dduugg) [inline README.md]: They should still be there – I'll update the PR if that isn't the case.

Comment (dduugg) [inline README.md]: I'm still seeing the correct issue counts: https://github.com/dduugg/cgm-remote-monitor/blob/034bcefc0f477d48b51538be2ff8b661b548a5d3/README.md

Comment (jasoncalabrese) [inline README.md]: oh, I see you just cleaned up the aliases

**Comment by bewest** [comment]
:+1:
